apps/OboeTester/app/src/main/cpp/cpu/AudioWorkloadTest.h

```diff
@@ -148,7 +149,10 @@ class AudioWorkloadTest : oboe::AudioStreamDataCallback {
         mAlternateNumVoices = alternateNumVoices;
         mAlternatingPeriodMs = alternatingPeriodMs;
         mStartTimeMs = 0;
-        mCallbackStatistics.clear();
+        {
+            std::lock_guard<std::mutex> lock(mStatisticsMutex);
```

Comment:
nit, just `std::lock_guard lock<mStatisticsMutex>` should work

Reply:
every single other mutex in oboe looks like this. Leaving as is